### ISSUE TITLE
Commonmark Support

### ISSUE THREAD

**Issue by v3ss0n** [trentm/python-markdown2#181]
As markdown is now Standardized by CommonMark team , this should support commonmark.org.

Reference implementation is already in python so should be easy to support.

**Comment by v3ss0n** [comment]
is this project dead?

**Comment by nicholasserra** [comment]
No, just need to catch up. I'll do some reviewing this weekend. Thanks!

**Comment by nicholasserra** [comment]
PRs welcome :)

**Comment by ChingxWando** [comment]
Hey there, do we support attribute lists at the moment?

**Comment by nicholasserra** [comment]
Not sure, to be honest. Maybe as an extra. If you investigate and find out that the answer is no, open an issue or pull request. Thanks!

**Comment by v3ss0n** [comment]
for web developer we no longer use serverside markdown , all render at frontend now.
